src/stories/actionbuttonV2.stories.js

```diff
@@ -0,0 +1,159 @@
+import { html } from 'lit-html';
+import '../components/atoms/ActionButtonV2/cod-action-button-v2';
+import '../components/atoms/Icon/cod-icon';
+import IconStory from './icon.stories';
+
+export default {
+  component: 'cod-action-button-v2',
+  title: 'Components/Atoms/ActionButtonV2',
+  // 👇 Creates specific argTypes
+  argTypes: {
+    buttonColor: {
+      options: [
+        'btn-outline-primary',
+        'btn-outline-secondary',
+        'btn-outline-success',
+      ],
+      control: 'select',
+    },
+    icon: IconStory.argTypes.icon,
+    title: {
+      control: 'text',
+    },
+    body: {
+      control: 'text',
+    },
+  },
+  args: {
+    buttonColor: 'btn-outline-primary',
+    title: 'Do Something',
+    body: 'Like click on this card',
+    icon: 'house-fill',
+  },
+};
+
+// Template
+const Template = (args) => {
+  const aButton = document.createElement('cod-action-button-v2');
+  aButton.setAttribute('btn-color', args.buttonColor);
+  aButton.setAttribute('icon', args.icon);
+
+  // Create a title slot
+  const titleSlot = document.createElement('h4');
+  titleSlot.setAttribute('slot', 'title');
+  titleSlot.innerText = args.title;
+  aButton.appendChild(titleSlot);
+
+  // Create a body slot
+  const bodySlot = document.createElement('p');
+  bodySlot.setAttribute('slot', 'body');
+  bodySlot.innerText = args.body;
+  aButton.appendChild(bodySlot);
+
+  aButton.setAttribute('href', 'https://example.com');
+  return aButton;
+};
+
+export const ActionButtonV2 = Template.bind({});
+
+export const ActionButtonV2RichBody = () => html`
+  <div style="width: 300px; height: 300px">
+    <cod-action-button-v2
+      btn-color="btn-outline-primary"
+      icon="house"
+      href="https://example.com"
+      target="_blank"
+    >
+      <h4 slot="title">Do Something</h4>
+      <p slot="body">
+        Anything can go inside an action button but it's best to keep to simple
+        text.
+      </p>
+      <img
```

Comment:
Would you include the image in the slot? That will let us test if it's handling more than just text in a reasonable way.